phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -1052,16 +1057,20 @@ private static void deletePriorSequences(long ts, Connection globalConn) throws
         }
         rs.close();
     }
-    
+
     protected static void initSumDoubleValues(byte[][] splits, String url) throws Exception {
-        ensureTableCreated(url, "SumDoubleTest", splits);
+        initSumDoubleValues("SumDoubleTest", splits, url);
+    }
+
+    protected static void initSumDoubleValues(String tableName, byte[][] splits, String url) throws Exception {
+        ensureTableCreated(url, tableName, "SumDoubleTest", splits);
```

Comment:
Minor nit: Define a static constant for SumDoubleTest like other table ddl types.